packages/bulk-edit/README.md

```diff
@@ -0,0 +1,30 @@
+<div align='center'>
+
+<br />
+
+<img src='https://raw.githubusercontent.com/eclipse-theia/theia/master/logo/theia.svg?sanitize=true' alt='theia-ext-logo' width='100px' />
+
+<h2>ECLIPSE THEIA - BULK EDIT EXTENSION</h2>
+
+<hr />
+
+</div>
+
+## Description
+
+The `@theia/bulk-edit` extension contributes an `bulk-edit` widget to the application that displays WorkspaceEdits to end-users.
```

Comment:
Should we instead describe the contributed widget as the _Preview Refactor_?

```suggestion
The `@theia/bulk-edit` extension contributes an `preview refactor` widget to the application that displays WorkspaceEdits to end-users.
```

Reply:
Fixed